### ISSUE TITLE
Add checks in auth generator that each step hasn't already been performed

### ISSUE THREAD

**Issue by cannikin** [redwoodjs/redwood#550]
For example if `import` statements have already been injected into the graphql file, don't try to add them again.

**Comment by jtoar** [comment]
@cannikin was what you had in mind with this different from #1480?

**Comment by cannikin** [comment]
Yep, although that only covers one of the steps (overwriting the auth.js file). I'm not sure that ALL other cases are covered right now...

I know there are checks to not insert a Page route if it's already there, and I think another to skip adding the `scaffold.css` file and `import` into `App.js`. We should probably take an inventory of everything all of the generators do and make sure that that nothing else doubles up, even if you run with `--force`

**Comment by jtoar** [comment]
Related: https://github.com/redwoodjs/redwood/issues/2685.
